docs/wiki/01_STUDY_PARAMETERS_SETUP/01_Study_Variables/PscID.md

```diff
@@ -0,0 +1,75 @@
+# PSCID (Project Study Center ID)
+
+The PSCID is the main participant identifier in Loris. With one ID assigned to each participant per project. It serves the purposes of this this this and this.
```

Comment:
technically not per project but per instance, if you are thinking of CBIG, we just know there are project overlaps and have a system to merge these participants.

Reply:
oh and I love the purpose